doc/user/administrator/troubleshooting/pipelines/index.md

```diff
@@ -0,0 +1,962 @@
+
+![rerun-galaxy-job.png][]
+
+What you should check for when rerunning the job is whether the error is reproducible, or perhaps there is a different error now showing up.
+
+## 5.2. Rerunning jobs from command-line
+
+**DANGER: Using the instructions in this method (rerunning the `tool_script.sh` file) *will* overwrite the previously generated files by this tool in Galaxy. Please only do this for tools that have errored and where you are certain you do not need previously-generated output files.**
+
+**Use this method only as a last resort and at your own risk.**
+
+If rerunning from the Galaxy UI does not give any clues as to what's going on, as a last resort you can also rerun from the command-line using the same environment as what Galaxy used to load tool dependencies.
+
+To do this, first find and change to the job working directory (as described in section [(4.2)][section-4.2]). For example, for job id `3362` we would change to:
+
+```
+cd database/jobs_directory/003/3362
+```
+
+Now, in here lets look at the `tool_script.sh` file:
+
+```
+#!/bin/bash
+...
+
+. /export/tool_deps/_conda/bin/activate '/export/tool_deps/_conda/envs/__refseq_masher@0.1.1' > conda_activate.log 2>&1
+...
+
+ln -s "/irida/sequence-files/1/2/SRR1952908_1.fastq" "SRR1952908_1.fastq" && ln -s "/irida/sequence-files/2/2/SRR1952908_2.fastq" "SRR1952908_2.fastq" &&  refseq_masher -vv contains --output refseq_masher-contains.tab --output-type tab --top-n-results 0 --parallelism "${GALAXY_SLOTS:-1}" --min-identity 0.9 --max-pvalue 0.01 "SRR1952908_1.fastq" "SRR1952908_2.fastq"
+```
+
+This file first tries to load up the tool dependencies (`. /export/tool_deps/_conda/bin/activate ...`). Then, this runs the actual command to produce the results.
+
+If you are on a machine that has access to the same conda environment (has access to `/export/tool_deps/_conda/bin/activate ...`), then you could try executing this script yourself (or parts of this script).
+
+```bash
+tool_script.sh
+```
+
+This could help give you insight into exactly why the specific tool is failing. However, you may have to modify the script to get it to work properly.
+
+**DANGER: Running `tool_script.sh` *will* overwrite previously generated files by this Galaxy job. Please only do this on jobs you are certain you do not need the output files for anymore.**
+
+# 6. Examples
+
+To tie everything together, let's work through troubleshooting a few example pipeline errors in IRIDA.
+
+## 6.1. Assembly and Annotation Pipeline `tbl2asn` error
+
+### 6.1.1. Input data
+
+I submitted the data from the [IRIDA Sample Data][irida-sample-data] download. Specifically, a sample created samples from the fastq files in `miseq-run/Data/Intensities/BaseCalls/08-5578*.fastq.gz`.
+
+![assembly-pipeline-default.png][]
+
+### 6.1.2. Pipeline error
+
+After a while or running, the pipeline encountered an error:
```

Comment:
Think you have a typo here, kinda oddly worded. "After a while or running"

Reply:
Fixed in c9fa7a06b31735e68f7a20b0706257e4966facae